### ISSUE TITLE
Checklist aanvullen

### ISSUE THREAD

**Issue by Thamerdal** [bliekp/BrewBuddy#21]
De geprinte checklist is een handig hulpmiddel tijdens het brouwen. Het zou fijn zijn als er stappen uitgehaald kunnen worden, toegevoegd kunnen worden en verschoven kunnen worden.

**Comment by ArjanvdL** [comment]
Ik heb me zojuist geregistreerd om exact dit voorstel te doen, maar je was me net een minuutje voor!
Momenteel print ik de checklist die ik dan vervolgens met handgeschreven toevoegingen aanpas. Zou heel fijn zijn als ik dit in brouwhulp zelf kan doen en dat hij dat voor de volgende keer onthoudt.

**Comment by Thamerdal** [comment]
Er zijn momenten in het brouwproces waarop je bijvoorbeeld een tussentijdse meting kunt doen. Volume, pH etc. In je brouwsoftware zie je soms aan het einde van je brouwdag dat je meer dingen had kunnen invullen in het scherm "Metingen op de brouwdag". Het gaat vaak om kleine reminders. Kan ook een tijds- of temperatuursaanduiding zijn bij whirlpoolen.
Alles wat in de checklist onder "Koelen" valt mag wat mij betreft dan weer vervallen, dat is voor mij zo klaar als een klontje.

**Comment by Thamerdal** [comment]
Nee, je wil juist een compacte checklist. Langer is niet perse beter.
Het gaat er juist om dat je flexibel bent. 

Als het anders te lastig gaat worden; wat een (tissen)oplossing zou kunnen zijn is een export van de checklist naar bijv Excel. 
Dan kun je daar verder zelf je aanpassingen doen.

**Comment by ArjanvdL** [comment]
Een aantal van de gegevens die je in kunt vullen in 'metingen op de brouwdag' missen inderdaad een meetpunt in de checklist. Waar het mij vooral om gaat echter is dat ik voor low oxygen brewing mijn maisch en spoelwater eerst voorkook en dan weer terugkoel, en aan het spoelwater ook antioxidanten toevoeg. Ook mag van mij alles over wort beluchten, filterdoeken en gist hydrateren eruit. Ik ga er vanuit dat andere brouwers andere items hebben. Daarom lijkt het me handig als het aanpasbaar is, wellicht via een xml bestand of iets dergelijks

**Comment by ingoogni** [comment]
Een checklist is een reflectie van een onderliggend proces. Als je een ander proces volgt klopt het inderdaad niet meer en het betekend ook dat je data behorende bij dat proces ook niet kunt opslaan.

De onderliggende, interessante, vraag is dan of we BrewBuddy zo kunnen aanpassen dat je je eigen proces(sen) kunt aanmaken/modelleren of dat we het aantal mogelijke processen kunnen uitbreiden en de volgorde daarvan aanpasbaar kunnen maken. Dat lijkt me op dit moment echt een brug te ver.

**Comment by Thamerdal** [comment]
Het is goed om een mooie standaard checklist te hebben, en het zou fijn zijn om daar tekstueel regels aan toe/tussen te kunnen voegen en verwijderen. Zo plat zou ik het graag zien.
Aanmaken van processen gaat m.i. veel te ver.
